drbd-formula.spec

```diff
@@ -60,7 +57,6 @@ cp -R examples %{buildroot}%{fdir}/states/%{fname}/%{ftemplates}/
 cp -R form.yml metadata.yml pillar.example README.md %{buildroot}%{fdir}/metadata/%{fname}
 
 %files
-%defattr(-,root,root,-)
 %if 0%{?sle_version} < 120300
```

Comment:
As commented, I would take advantage of these fixes and remove the `< 120300` condition. We don't build anymore for versions less than SLE12SP3, and maintenance has been a bit picky in these things.
The next would be just fine:
```
%doc README.md
%license LICENSE
```
But well, we can do this any other time

Reply:
Got it, i will fix it this time.